docs/server-side/v2/policy/create-template-via-api.mdx

```diff
@@ -442,14 +442,14 @@ curl --location --request POST 'https://api.100ms.live/v2/templates' \
+| Name            | Type     | Description                                                                                                                                                                                                                                                                                                                                                   | Required |
+| --------------- | :------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| name            | `string` | Alias for the role associated with a particular template.                                                                                                                                                                                                                                                                                                     | No       |
+| publishParams   | `object` | Publish parameters for this role. <br/><br/>Object of type [publishParams](#publishparams-object) with the collection of objects such as `allowed`, `audio`, `video`, `screen`, etc                                                                                                                                                                           | No       |
+| subscribeParams | `object` | Subscribe parameters for this role. <br/><br/>Object of type [subscribeParams](#subscribe-object) with a mix of arguments and objects like `subscribeToRoles`, `maxSubsBitRate`, & `subscribeDegradation`                                                                                                                                                     | No       |
+| permissions     | `object` | Permissions for this role. <br/><br/>Object of type [permission](#permissions-object) with collection of objects like `browserRecordings`, `rtmpDestinations`, & `hlsDestinations`.                                                                                                                                                                           | No       |
+| priority        | `int`    | Priority of the role for subscribing/publishing. <br/><br/>Priority will determine the order in which the roles will be degraded. A lower number represents a higher priority.                                                                                                                                                                                | No       |
```

Comment:
this needs to have allowed list of values. i believe it's 1-5, but please confirm

Reply:
added